opentelemetry-cpp.yaml

```diff
@@ -0,0 +1,49 @@
+package:
+  name: opentelemetry-cpp
+  version: 1.17.0
+  epoch: 0
+  description: The OpenTelemetry C++ Client
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - abseil-cpp-dev
+      - benchmark-dev
+      - busybox
+      - c-ares-dev
+      - cmake
+      - gcc
+      - glibc-dev
+      - grpc-dev
+      - gtest-dev
+      - icu-dev
+      - make
+      - openssl-dev
+      - pkgconf
+      - protobuf-dev
+      - re2-dev
+      - systemd-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/open-telemetry/opentelemetry-cpp
+      tag: v${{package.version}}
+      expected-commit: fa0a5200ddbe9ae6f487a228e658aedc102dab56
+
+  - runs: |
+      mkdir build
+      cd build
+      cmake -DCMAKE_POSITION_INDEPENDENT_CODE=ON -DBUILD_SHARED_LIBS=ON -DWITH_OTLP_GRPC=ON -DWITH_ABSEIL=ON ..
+      cmake --build . --target all
+      cmake --install . --prefix ${{targets.contextdir}}/usr
+
+  - uses: strip
+
```

Comment:
Any docs/infodirs we can split into subpackages, development files. etc?

Reply:
We have pipelines in melange that can be used to split things away: https://github.com/chainguard-dev/melange/tree/main/pkg/build/pipelines/split

Typically, we split whatever we can. I.E., those headers (the `.h` files) would be split away with `split/dev` and docs would be split with `split/manpages` and `split/info` if an info directory exists. It looks like in your case, you just need to create a `-dev` subpackage that uses the `split/dev` pipeline

Reply:
💯 exactly what I wanted. Thanks! 🎉